app/controllers/press-releases/press-release.js

```diff
@@ -59,15 +62,18 @@ export default class PressReleasesPressReleaseController extends Controller {
   *publish(publicationDate) {
     let publicationEvent = yield this.snapshot.pressRelease.publicationEvent;
     if (!publicationEvent) {
-      publicationEvent = this.store.createRecord('publication-event', {
+        publicationEvent = this.store.createRecord('publication-event', {
         plannedStartDate: publicationDate,
         pressRelease: this.snapshot.pressRelease
       });
     } else {
-      publicationEvent.plannedStartDate = publicationDate;
+      if(!publicationEvent) {
+        publicationEvent.started = publicationDate;
+      } else {
+        publicationEvent.plannedStartDate = publicationDate;
+      }
```

Comment:
Ik begrijp niet wat je hier nog probeert te doen?
In het geval er nog geen publication-event is, wordt er een nieuw aangemaakt in de `if`-branch op lijn 64.
Anders moet gewoon de `plannedStartDate` ingesteld worden op het bestaande publication-event. De frontend moet nooit een wijziging doen aan de `started` date van een publication-event.

Reply:
ik had dit weggedaan na je eerste uitleg, ik denk dat het door mergen terug er in is gekomen.